test/testsuites/opsrccreationtests.go

```diff
@@ -23,34 +24,21 @@ func OpSrcCreation(t *testing.T) {
 func testOperatorSourceGeneratesExpectedObjects(t *testing.T) {
 	// Get test namespace
 	namespace, err := test.NewTestCtx(t).GetNamespace()
-	if err != nil {
-		t.Errorf("Could not get namespace: %v", err)
-	}
+	assert.NoError(t, err)
 
 	// Check for the CatalogSourceConfig and it's child resources.
 	err = helpers.CheckCatalogSourceConfigAndChildResourcesCreated(test.Global.Client, helpers.TestOperatorSourceName, namespace, namespace)
-	if err != nil {
-		t.Error(err)
-	}
+	assert.NoError(t, err)
 
 	// Then check for the CatalogSource.
 	resultCatalogSource := &olm.CatalogSource{}
 	err = helpers.WaitForResult(test.Global.Client, resultCatalogSource, namespace, helpers.TestOperatorSourceName)
-	if err != nil {
-		t.Error(err)
-	}
+	assert.NoError(t, err)
 
 	// Check that the CatalogSource has the expected labels.
 	labels := resultCatalogSource.GetLabels()
 	groupGot, ok := labels[helpers.TestOperatorSourceLabelKey]
 
-	if !ok || groupGot != helpers.TestOperatorSourceLabelValue {
-		t.Errorf(
-			"The created CatalogSource %s does not have the right label[%s] - want=%s got=%s",
-			resultCatalogSource.Name,
-			helpers.TestOperatorSourceLabelKey,
-			helpers.TestOperatorSourceLabelValue,
-			groupGot,
-		)
-	}
+	assert.Equal(t, true, ok)
+	assert.Equal(t, groupGot, helpers.TestOperatorSourceLabelValue)
```

Comment:
This is the signature of `assert.Equal`
```
Equal(t TestingT, expected, actual interface{}, msgAndArgs ...interface{}) bool {
```

Suggestion: By convention we specify expected first and then actual. `groupGot` is actual and I think `helpers.TestOperatorSourceLabelValue` is your expected here.

Reply:
@anik120 please ensure that you follow this convention in all places.